main.tf

```
@@ -126,31 +126,31 @@ module "eks" {
   #   instance_types = ["m6i.large", "m5.large", "m5n.large", "m5zn.large"]
   # }
 
-  # eks_managed_node_groups = {
-  #   one = {
-  #     name         = var.eks_nodeGroup
-  #     desired_size = 1
-  #     min_size     = 0
-  #     max_size     = 10
-
-  #     instance_types = ["t3.large"]
-  #     capacity_type  = "SPOT"
-  #     iam_role_additional_policies = {
-  #       AmazonEBSCSIDriverPolicy = "arn:aws:iam::aws:policy/service-role/AmazonEBSCSIDriverPolicy",
-  #       SecretsManagerReadWrite  = "arn:aws:iam::aws:policy/SecretsManagerReadWrite"
-  #     }
-  #   }
-  #   # ,
-  #   # two = {
-  #   #   name         = "seqera"
-  #   #   desired_size = 1
-  #   #   min_size     = 0
-  #   #   max_size     = 10
-
-  #   #   instance_types = ["t3.large"]
-  #   #   capacity_type  = "SPOT"
-  #   # }
-  # }
+  eks_managed_node_groups = {
+    one = {
+      name         = var.eks_nodeGroup
+      desired_size = 1
+      min_size     = 0
+      max_size     = 10
```

Spot recommended us to have these
```suggestion
      desired_size = 0
      min_size     = 0
      max_size     = 0
```

---

I swapped this to a max_size of 1 (0) is not valid.

Where did they recommend this? Their docs are all over the place and I couldn't find this piece